src/ape/api/transactions.py

```diff
@@ -294,6 +294,25 @@ def validate_txn_hash(cls, value):
     def call_tree(self) -> Optional[Any]:
         return None
 
+    @cached_property
+    def debug_logs_typed(self) -> List[Tuple[Any]]:
+        """Return any debug log data outputted by the transaction"""
+        return []
+
+    @cached_property
+    def debug_logs_lines(self) -> List[str]:
+        """
+        Return any debug log data outputted by the transaction as strings suitable for printing
+        """
+        return [" ".join(map(str, ln)) for ln in self.debug_logs_typed]
+
+    def show_debug_logs(self):
+        """
+        Output debug logs to logging system
+        """
+        for ln in self.debug_logs_lines:
+            logger.info("[DEBUG-LOG] " + ln)
```

Comment:
nit: i love f-strings, they are so readable

```suggestion
            logger.info(f"[DEBUG-LOG] {ln}")
```